### PR TITLE
Navweb: add option to show all places

### PR DESCRIPTION
**Pull request by SNoiraud**
Fixes #012315

### CODE REVIEW

File: gramps/plugins/webreport/narrativeweb.py [hunk 1]
```diff
@@ -2387,6 +2387,11 @@ def __add_advanced_options_2(self, menu):
             _('Whether or not to include the places Pages.'))
         addopt("inc_places", inc_places)
 
+        inc_uplaces = BooleanOption(_('Include unused places pages'), False)
```
Comment (Nick-Hall) [inline gramps/plugins/webreport/narrativeweb.py:2390]: "place pages" instead of "places pages"

File: gramps/plugins/webreport/narrativeweb.py [hunk 2]
```diff
@@ -2387,6 +2387,11 @@ def __add_advanced_options_2(self, menu):
...
+        inc_uplaces.set_help(
+            _('Whether or not to include the unused places Pages.'))
```
Comment (Nick-Hall) [inline gramps/plugins/webreport/narrativeweb.py:2392]: "place pages" instead of "places Pages"

**Review by Nick-Hall — CHANGES_REQUESTED** [review]
A couple of minor changes to the text.

File: gramps/plugins/webreport/narrativeweb.py (hunk 2 above)
Comment (SNoiraud) [reply gramps/plugins/webreport/narrativeweb.py:2392]: Done

File: gramps/plugins/webreport/narrativeweb.py (hunk 1 above)
Comment (SNoiraud) [reply gramps/plugins/webreport/narrativeweb.py:2390]: Done

File: gramps/plugins/webreport/narrativeweb.py [hunk 3]
```diff
@@ -2382,10 +2382,15 @@ def __add_advanced_options_2(self, menu):
...
+            _('Whether or not to include the place Pages.'))
         addopt("inc_places", inc_places)
 
-        inc_sources = BooleanOption(_('Include source pages'), False)
+        inc_uplaces = BooleanOption(_('Include unused place pages'), False)
+        inc_uplaces.set_help(
+            _('Whether or not to include the unused place Pages.'))
+        addopt("inc_uplaces", inc_uplaces)
+
+        inc_sources = BooleanOption(_('Include sources pages'), False)
```
Comment (Nick-Hall) [inline gramps/plugins/webreport/narrativeweb.py:2393]: This should be changed back to "source pages".

Comment (SNoiraud) [reply gramps/plugins/webreport/narrativeweb.py]: I hope this is solved.

**Pull request merged**